### DISCUSSION TITLE
Input autocomplete

### DISCUSSION THREAD

**Discussion by adjsky** [scalar/scalar#1762] [General]
Hello again!

When testing authentication i've found that if you type a value into the authentication input and press Enter, the value'll be saved and auto-filled later (which is generally ok), however, some browsers by default change input color to yellow or blue, which looks pretty awful:

![image](https://github.com/scalar/scalar/assets/49305219/32fa997a-f4c2-4c1a-ab39-72ee43fc4c0d)

I could open a PR with fixes, so input doesn't get recolored, like this:

![image](https://github.com/scalar/scalar/assets/49305219/e03f1043-ed73-4b29-b588-54b96a59cced)

But I'd like to know your thoughts on it first, in case this is expected behavior.

**Comment by antlio** [comment]
good catch, i'm all for it. what do you think? @cameronrohani

**Reply by marclave** [reply]
lets do it! @adjsky feel free to make a PR for this ✨
